src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -35,24 +37,44 @@ export class AddBulkConcepts extends Component {
     this.state = {
       conceptIds: '',
       openResultModal: false,
+      otherSelected: false,
     };
     this.invalidConceptIds = [];
-    this.sourceUrl = 'orgs/CIEL/sources/CIEL/concepts/';
+    this.sourceUrl = 'orgs/CIEL/sources/CIEL/';
   }
 
   componentDidMount() {
-    this.props.fetchCielConcepts();
+    this.props.getConceptSources();
+    this.props.fetchSourceConcepts(this.sourceUrl);
```

Comment:
@ozezzy, kindly destructure props here.

Reply:
Destructuring getConceptSources and fetchSourcesConcepts here gives linting error because of line 11. They are already declared in the scope.